src/smexperiments/trial_component.py

```diff
@@ -94,21 +98,6 @@ def load(cls, trial_component_name, sagemaker_boto_client=None):
         )
         return trial_component
 
-    @classmethod
-    def create(cls, trial_component_name, display_name=None, sagemaker_boto_client=None):
```

Comment:
Why do we want to remove it? The Eureka SDE can still be used within SM environment where direct call to CreateTC is allowed, right?

Reply:
the sagemaker environment limitation is enforced by account id (see accountCanCallFromNonSageMakerContext) so it appears any other context info where this sdk is calling from is irrelevant.  

There are other options such as logging a warning, raising an exception, or just adding documentation and comments.

Reply:
even if user can create tc by tracker (say in sm notebook), but why we choose to close this option for user? i.e. creating from TrailComponent class.

Reply:
we want to remove the create option because:
* all usage of create TC from anywhere (this sdk or API) is currently blocked (unless added to internal whitelist)
* customers trying to use this method are confused by the error message (see AML-52445)

Reply:
hmm, meaning we need to update the sample notebook either ....

Reply:
we need to remove the create in Tracker too? Is this blocking permanent?

Reply:
looks like the tracker is retrieving an existing TC by name or source arn, not actually creating a TC.  

The sample notebook is not using TrialComponent.create

Reply:
no, the tacker.create is calling trialComponent.create

https://github.com/aws/sagemaker-experiments/blob/master/src/smexperiments/tracker.py#L145

Reply:
This should work when used in a SageMaker notebook. Assuming that this is the case, then

- Let's go ahead with this change.
- Fix the problem in SageMaker notebooks ASAP, then reinstate this functionality.